### ISSUE TITLE
Janky Colour Swapping

### ISSUE THREAD

**Issue by sabotawsh** [kennethcachia/background-check#114]
Hello!

I have the plugin working, but it's not consistent between images. Sometimes it works and sometimes it doesn't. There aren't any dev errors so I'm struggling to correct this.

Any idea?

**Comment by XavierF** [comment]
Was having the same issue too and was driving me crazy but this was happening on page load
with the same image. I went and set debug: true and reloaded the page and found this: 
<img width="793" alt="Screen Shot 2019-10-28 at 6 03 04 PM" src="https://user-images.githubusercontent.com/419588/67729172-4bed1d80-f9ad-11e9-9beb-c1ad63025112.png">
reading the documentation background check stops working or kills the process after 500 ms...
which is it's default setting. In the js I added maxDuration: 1000
seems to work fine now

Edit: It's also mindful to keep img file sizes as small as possible for faster loading.
The issue on my end is I'm using a background img in a hero div on a dev server so I will be reducing image file sizes to improve performance before I go live.
